aws/common/s3.tf

```
@@ -24,7 +25,7 @@ resource "aws_s3_bucket" "csv_bucket" {
 
   #tfsec:ignore:AWS077 - Versioning is not enabled
   logging {
-    target_bucket = aws_s3_bucket.csv_bucket_logs.bucket
+    target_bucket = regex("^[^.]*", module.csv_bucket_logs.s3_bucket_domain_name)
```

You should be able to use the [module's `s3_bucket_id` output](https://github.com/cds-snc/terraform-modules/blob/24e4de7bbb94139504fed050bb26f5ee55e105d5/S3_log_bucket/outputs.tf#L1-L4) for this:
```suggestion
    target_bucket = module.csv_bucket_logs.s3_bucket_id
```

---

100% correct - forgot that aws treats id and bucket name as the same thing. Fixed.